gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/EmergePlugin.kt

```diff
@@ -243,13 +247,19 @@ class EmergePlugin : Plugin<Project> {
       }
     }
 
-    appProject.tasks.register(taskName, InitializeReaper::class.java) {
+    appProject.tasks.register(uploadAabTaskName, InitializeReaper::class.java) {
       it.group = EMERGE_TASK_GROUP
-      it.description = "Confirms Reaper is initialized and uploads an AAB for variant ${variant.name} to Emerge."
+      it.description = "Uploads an AAB for variant ${variant.name} to Emerge."
```

Comment:
Might add that this also adds Reaper to the AAB, important note!

Reply:
done!